.travis.yml

```diff
@@ -14,4 +14,4 @@ before_script:
   - brew link bdw-gc
 script:
   - java -version
-  - sbt 'rtlib/publishLocal' 'nscplugin/publishLocal' "demoNative/run"
+  - sbt 'rtlib/publishLocal' 'nscplugin/publishLocal' 'compile'
```

Comment:
I think ultimately we need both `demoNative/run` and `sandbox/run` there too, not just `compile`.